### ISSUE TITLE
Embedded widgets

### ISSUE THREAD

**Issue by plhrja** [announcekitapp/announcekit-angular#3]
How are you to construct embedded widgets? As far as I can see, the component itself does not take the element-selector where the widget should be embedded as input.

**Comment by namirali** [comment]
Yes it does not take and need any selector as the component creates an element and embed the widget into it. You can check the sample here https://codesandbox.io/s/announcekit-angular-u4nxq?file=/src/app/app.component.html
